lib/angular/controller.js

```diff
@@ -0,0 +1,22 @@
+var CONSTANTS = require('./constants');
+
+/**
+ *
+ * Controller for listing Workorders
+ *
+ * @param {Mediator} mediator
+ * @param {MapMediatorService} mapMediatorService
+ * @constructor
+ */
+function MapController(mediator, mapMediatorService) {
+  var self = this;
+
+  self.workorders = [];
+  self.resultMap = {};
+
+  mapMediatorService.listWorkorders().then(function(results) {
```

Comment:
Small thing, should be `function(workorders)` instead of `function(results)`. Might be a bit confusing otherwise.